spec/controllers/api/v1/patients_controller_spec.rb

```diff
@@ -1,15 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::PatientsController, type: :controller do
-  describe 'POST sync: send data from device to server;' do
+  let(:model) { Patient }
 
-    describe 'creates new patients' do
+  let(:build_payload) { lambda { build_patient_payload } }
+  let(:build_invalid_payload) { lambda { build_invalid_patient_payload } }
+  let(:update_payload) { lamda { |record| updated_patient_payload record } }
 
-      it 'returns 400 when there are no patients in the request' do
-        post(:sync_from_user, params: { patients: [] })
-        expect(response.status).to eq 400
-      end
+  let(:number_of_schema_errors) { 2 + invalid_record['phone_numbers'].count }
```

Comment:
I definitely don't know what this is for. Where is `invalid_record`?

Reply:
We have extracted `invalid_record` into a `let` above this, and renamed this to `number_of_schema_errors_in_invalid_payload` as well.